helm-charts/falcon-sensor/templates/container_deployment_webhook.yaml

```diff
@@ -98,6 +99,24 @@ spec:
           mountPath: /run/azure.json
           readOnly: true
       {{- end }}
+      {{- end }}
+      {{- if not .Values.container.azure.enabled }}
+      {{- if .Values.container.gcp.enabled }}
+      initContainers:
+      - name: {{ include "falcon-sensor.name" . }}-init-container
+        image: "gcr.io/google.com/cloudsdktool/cloud-sdk:alpine"
```

Comment:
Just to be sure, is the `cloud-sdk` image required, or can it be any container image with curl?

Reply:
it is enough any image with curl

Reply:
Cool. Thanks! Wanted to be sure that that is the case.